app/javascript/components/organization/RegisterForm.jsx

```diff
@@ -15,7 +15,7 @@ class RegisterForm extends React.Component {
       currentStep: 1,
       email: "",
       contactPhoneNumber: "",
-      selectedProfileFile: {},
+      selectedProfileFile: null,
```

Comment:
@ethanlee16 Let's remove `validateField = (fieldName, value) => {` while we're at it since I see a different validateField is being used

Reply:
Also i couldn't add a comment on line 35 sad, but that's what this comment is referring to

Reply:
Done!